overlord/snapstate/aliasesv2.go

```diff
@@ -342,3 +342,35 @@ func checkAliasesConflicts(st *state.State, snapName string, candAutoDisabled bo
 	}
 	return nil, nil
 }
+
+// disableAliases returns newAliases corresponding to the disabling of curAliases, for manual aliases that means removed.
+func disableAliases(curAliases map[string]*AliasTarget) (newAliases map[string]*AliasTarget) {
+	newAliases = make(map[string]*AliasTarget, len(curAliases))
+	for alias, curTarget := range curAliases {
+		if curTarget.Auto != "" {
+			newAliases[alias] = &AliasTarget{Auto: curTarget.Auto}
+		}
+	}
+	return newAliases
+}
+
+// pruneAutoAliases returns newAliases by dropping the automatic aliases autoAliases from curAliases, used as the task prune-auto-aliases to handle transfers of automatic aliases in a refresh.
```

Comment:
Per our call this week, I think it'd make sense to disable manual aliases in this case as well. It's not a big deal to move it forward like this if you'd prefer, since we can easily tune the behavior later as we learn more, but it feels slightly surprising to have only the auto ones disabled.